.circleci/continue_config.yml

```diff
@@ -516,6 +516,23 @@ jobs:
 workflows:
   version: 2.1
 
+  lint_docs_only:
+    when:
+      and:
+        - <<pipeline.parameters.docs_change>>
+        - not: <<pipeline.parameters.code_change>>
+        - not: <<pipeline.parameters.release_kedro>>
+        - not: <<pipeline.parameters.run_hourly>>
+        - not: <<pipeline.parameters.run_nightly>>
+    jobs:
+      - lint:
+          matrix:
+            parameters:
+              python_version: ["3.7", "3.8", "3.9", "3.10"]
+      - all_circleci_checks_succeeded:
+          requires:
+            - lint
+
   build_code_and_docs:
```

Comment:
This doesn't build docs anymore right? Again might be good to rename for clarity.

Reply:
Oh yep, on it